web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1712,6 +1712,18 @@ export const formatters: Formatters = {
     format: ({ updated_by }) =>
       `User [${updated_by}] disabled External Audit Storage`,
   },
+  [eventCodes.SPIFFE_SVID_ISSUED]: {
+    type: 'spiffe.svid.issued',
+    desc: 'SPIFFE SVID Issued',
+    format: ({ user, spiffe_id }) =>
+        `User [${user}] requested SPIFFE SVID [${spiffe_id}]`,
+  },
+  [eventCodes.SPIFFE_SVID_ISSUED_FAILURE]: {
+    type: 'spiffe.svid.issued',
+    desc: 'SPIFFE SVID Issued Failure',
+    format: ({ user, spiffe_id }) =>
+        `User [${user}] failed to request SPIFFE SVID [${spiffe_id}]`,
```

Comment:
is it failed to request or failed to issue? I'm not sure which terminology makes most sense.

Reply:
Hm - yeah - I think failed to issue is clearer. I'll switch to that.

Reply:
65f6ef3813